sos/report/plugins/pcp.py

```diff
@@ -10,7 +10,6 @@
 
 from sos.report.plugins import Plugin, RedHatPlugin, DebianPlugin
 import os
-import os.path
```

Comment:
Hmm... why some CI job does not detect this automatically? There is a warning like "unused import specified"..?

Reply:
> We might also replace `os.path.isfile(path)` in https://github.com/sosreport/sos/blob/master/sos/report/plugins/jars.py#L90 but that wont gain much, since `os.path` is used in the plugin in either way as well.

Can't use the helpers there since `is_jar()` is a staticmethod.

> Hmm... why some CI job does not detect this automatically? There is a warning like "unused import specified"..?

I was a bit surprised this wasn't caught either by `flake8`, but I'm not sure why it isn't off hand.